src/Lean/Meta/ExprDefEq.lean

```diff
@@ -888,14 +888,15 @@ def checkAssignment (mvarId : MVarId) (fvars : Array Expr) (v : Expr) : MetaM (O
       let v ← instantiateMVars v
       CheckAssignment.checkAssignmentAux mvarId fvars hasCtxLocals v
 
-private def processAssignmentFOApproxAux (mvar : Expr) (args : Array Expr) (v : Expr) : MetaM Bool :=
+private partial def processAssignmentFOApproxAux (mvar : Expr) (args : Array Expr) (v : Expr) : MetaM Bool :=
```

Comment:
You should also be able to remove `partial` now that the typo was fixed.

Reply:
Right, thanks... I started out with `consumeMData` but noticed neither of them was used in other parts of the file, perhaps for performance purposes I assumed

Reply:
`consumeMData` was added after we wrote this file.